README.md

```diff
@@ -1563,10 +1563,6 @@ Feel free to talk with me on Twitter! <3
 
   Provides Big O notation benchmarking for Ruby.
 
-- [The Ruby Challenge](https://therubychallenge.com/)
-
-  Talk by Prem Sichanugrist @ Ruby Kaigi 2014.
```

Comment:
I found the talk on YT https://www.youtube.com/watch?v=aDeP7FGQBig. Since it's a YT link, I'm okay with adding it back (it's less likely to get removed easily).